docs/gbfs-validator.yaml

```diff
@@ -0,0 +1,187 @@
+openapi: 3.0.3
+info:
+  title: MobilityData GBFS Validator
+  version: 1.0.1
+  contact:
+    name: MobilityData
+    url: https://mobilitydata.org/
+    email: api@mobilitydata.org
```

Comment:
@emmambd We reused this email unless you think it should be a different address.

Reply:
@davidgamez No, this works! Anything relevant to @richfab I can forward accordingly.